resources/tx_curr-Measure.json

```diff
@@ -0,0 +1,57 @@
+{
+    "resourceType": "Measure",
+    "id": "TX-CURR",
+    "identifier": [
+        {
+            "use": "official",
+            "value": "TX_CURR"
+        }
+    ],
+    "version": "0.0.1",
+    "title": "Sample TX_CURR population selection criteria and evaluation",
+    "status": "draft",
+    "experimental": true,
+    "library": [
+        "Library/TX-CURR"
+    ],
+    "scoring": {
+        "coding": [
+            {
+                "code": "proportion"
```

Comment:
@gitcliff ,can you change this to the right measure scoring code, see https://www.hl7.org/fhir/valueset-measure-scoring.html

Reply:
@mozzy11 i have fixed it

Reply:
@mozzy11  is this now beautiful enough?

Reply:
yes @HerbertYiga , it lloks good. let me test and see